submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/ext/getSimpleByAccNo.kt

```diff
@@ -0,0 +1,6 @@
+package ac.uk.ebi.biostd.submission.ext
+
+import ac.uk.ebi.biostd.persistence.common.service.SubmissionQueryService
+import ebi.ac.uk.extended.mapping.to.toSimpleSubmission
+
+fun SubmissionQueryService.getSimpleByAccNo(accNo: String) = getExtByAccNo(accNo).toSimpleSubmission()
```

Comment:
Why is this an extension?

Reply:
This service comes from the persistence module so if we won't allow it in class we need to add the serialization module as a dependency on persistence. As that is not necessary at all. I Declared this an extension of main product (that already depends on the serialization model)